src/components/input.js

```diff
@@ -301,6 +305,31 @@ export const TextArea = ({ onChange, autosize = false, nativeOnChange = false, .
   }, props))
 }
 
+/**
+ * Utility function for wrapping text on periods.
```

Comment:
```suggestion
 * Adds a WBR element before each period which advises the browser that a line-break here is acceptable.
```

Reply:
@dmohs we actually decided to remove the function completely (https://broadinstitute.slack.com/archives/C01EHNUM73R/p1630528774002800)